manifests/params.pp

```diff
@@ -99,8 +99,6 @@
 
       if $postgresql::globals::postgis_package_name {
         $postgis_package_name = $postgresql::globals::postgis_package_name
-      } elsif $facts['os']['release']['major'] == '5' {
-        $postgis_package_name = 'postgis'
       } elsif $postgis_version and versioncmp($postgis_version, '2') < 0 {
```

Comment:
As I noted elsewhere: in EL 8 & EL 9 I didn't find a postgis package. For the official packages I see there's postgis32 and postgis33 (https://download.postgresql.org/pub/repos/yum/15/redhat/rhel-9-x86_64/) so the else condition isn't quite true either.

Reply:
I'm not really sure how we should move forward. It seems like there are no acceptance tests for this code and I don't know on which systems they work / are supposed to work. My goal was to simplify the conditions here and clean it up. I can raise a ticket about the mismatch.

Reply:
Sounds good to me.

Reply:
raised https://github.com/puppetlabs/puppetlabs-postgresql/issues/1522